gaas-ui/src/components/create-graph/graph-id-description.tsx

```diff
@@ -61,14 +63,25 @@ export default function GraphIdDescriptionInput(props: IProps): ReactElement {
                         "aria-label": "graph-description-input",
                     }}
                     value={descriptionValue}
+                    error={descriptionErrorHelperText.length > 0}
                     required
                     multiline
                     autoFocus
                     rows={5}
                     fullWidth
                     name="graph-description"
                     variant="outlined"
-                    onChange={(event) => onChangeDescription(event.target.value)}
+                    helperText={descriptionErrorHelperText}
+                    onChange={(event) => {
+                        const regex = new RegExp("^[a-zA-Z0-9 ]*$");
```

Comment:
Am I right that this isn't sanitisation but instead checking that the description matched our defined restrictions?

Reply:
that's correct